securing_apps/topics/overview/supported-protocols.adoc

```diff
@@ -2,6 +2,8 @@
 
 === Supported Protocols
 
+For protocols, {project_name} supports both OpenID Connect and SAML.
```

Comment:
Perhaps: "{project_name} supports both OpenID Connect and SAML protocols"? Or similar. No need for "For protocols" at the beginning?